src/client/versionChecking.js

```diff
@@ -11,7 +14,7 @@ module.exports = function (client, options) {
     let text = parsed.text ? parsed.text : parsed
     let versionRequired
 
-    if (text.translate && text.translate.startsWith('multiplayer.disconnect.outdated_')) { versionRequired = text.with[0] } else {
+    if (text.translate && (text.translate.startsWith('multiplayer.disconnect.outdated_') | text.translate.startsWith('multiplayer.disconnect.incompatible'))) { versionRequired = text.with[0] } else {
```

Comment:
Can you un-inline this if statement?

Reply:
Just un-inlined the if. Not sure why it was inlined to begin with.